pkg/utils/fakeServerUtils.go

```diff
@@ -7,35 +7,37 @@ import (
 )
 
 func MultitpleMockRequest(getResponses []string, postResponses []string, paths []string, sslVerificationActivated bool) *httptest.Server {
-	server := &httptest.Server{}
-	if sslVerificationActivated {
+	var server *httptest.Server
+	switch sslVerificationActivated {
+	case true:
 		server = httptest.NewTLSServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
```

Comment:
I notice now that the `http.HandlerFunc(....)` parameter can be extracted and stored in a variable before the switch